src/components/Footer.js

```diff
@@ -0,0 +1,39 @@
+import StyledFooter from './styled-components/StyledFooter';
+import StyledImage from './styled-components/StyledImage';
+import StyledUL from './styled-components/StyledUL';
+
+export default function AddProject1() {
+	return (
+		<StyledFooter>
+			<StyledUL variant="footer">
```

Comment:
Your StyledFooter is a \<ul> and your StyledUL is a \<ul> too... Did you want that? A ul that holds a ul, that holds li's ?! Mhhhh.... strange.... 🤔